source/deployment-options/deploying-with-puppet/wazuh-puppet-module/index.rst

```diff
@@ -45,23 +45,208 @@ This module installs and configures Wazuh agent and manager.
 Install a stack via Puppet
 --------------------------
 
-A stack of Wazuh manager, Wazuh dashboard, Wazuh indexer and Filebeat can be deployed using this manifest. See the Wazuh manager section for variables that can be used to configure the manager before deployment.
+Single Node:
+------------
+
+A single node stack of Wazuh manager, Wazuh dashboard, Wazuh indexer and Filebeat can be deployed using this manifest. See the Wazuh manager section for variables that can be used to configure the manager before deployment.
 
 Create a stack.pp file at ``/etc/puppetlabs/code/environments/production/manifests/`` and put the contents below. ``puppet-agent-node`` refers to the hostname or IP of the puppet agent:
 
 .. code-block:: console
 
-   node "puppet-agent-node" {
-    class { 'wazuh::manager':
-    }
-    class { 'wazuh::indexer':
-    }
-    class { 'wazuh::filebeat_oss':
-    }
-    class { 'wazuh::dashboard':
-    }
-   }
+      $discovery_type = 'single-node'
+      stage { 'certificates': }
+      stage { 'repo': }
+      stage { 'indexerdeploy': }
+      stage { 'securityadmin': }
+      stage { 'dashboard': }
+      stage { 'manager': }
+      Stage[certificates] -> Stage[repo] -> Stage[indexerdeploy] -> Stage[securityadmin] -> Stage[manager] -> Stage[dashboard]
+      Exec {
+      timeout => 0,
+      }
+      node "puppet-server" {
+      class { 'wazuh::certificates':
+        indexer_certs => [['node-1','127.0.0.1']],
+        manager_certs => [['master','127.0.0.1']],
+        dashboard_certs => ['127.0.0.1'],
+        stage => certificates,
+      }
+      }
+      node "puppet-aio-node" {
+      class { 'wazuh::repo':
+      stage => repo,
+      }
+      class { 'wazuh::indexer':
+        stage => indexerdeploy,
+      }
+      class { 'wazuh::securityadmin':
+      stage => securityadmin
+      }
+      class { 'wazuh::manager':
+        stage => manager,
+      }
+      class { 'wazuh::filebeat_oss':
+        stage => manager,
+      }
+      class { 'wazuh::dashboard':
+        stage => dashboard,
+      }
+      }
+
+Multi Node:
+-----------
+
+You can deploy a distributed stack with 1 ``Wazuh dashboard``, 1 ``Wazuh manager master``, 1 ``Wazuh manager worker`` and 3 ``Wazuh indexer`` nodes between 3 diferentes servers or VMs with this manifest.
+
+It is required to add within the manifest the IPs of the servers where each of the applications will be installed.
+
+This is the correspondence of the IPs with the puppet nodes described in the manifest:
+
+.. code-block:: console
```

Comment:
What is this? These are not terminal commands, right? We can use `none` instead

Reply:
Done: https://github.com/wazuh/wazuh-documentation/pull/6186/commits/3448299e91dcbc588dc4812a8b724b9deecd7bd5